postgres/osfamilymap.yaml

```diff
@@ -4,9 +4,10 @@
 
 Arch:
   conf_dir: /var/lib/postgres/data
+  data_dir: /var/lib/postgres/data
   prepare_cluster:
-    command: initdb -D /var/lib/postgres/data
-    test: test -f /var/lib/postgres/data/PG_VERSION
+    pgcommand: initdb -D
+    pgtestfile: PG_VERSION
```

Comment:
Since these values would exactly match the default ones, no need to override them here. Same for other distros below.

Reply:
Good point.  I'll fix that. thanks @vutny